source/util/AMDSErrorMonitor.h

```diff
@@ -0,0 +1,275 @@
+/*
```

Comment:
I started the AMDS project from scratch. So I did the same thing for AMDSnDIndex and whatnot.